platform/linuxbsd/godot_linuxbsd.cpp

```diff
@@ -72,18 +72,19 @@ int main(int argc, char *argv[]) {
 	char *ret = getcwd(cwd, PATH_MAX);
 
 	Error err = Main::setup(argv[0], argc - 1, &argv[1]);
+
 	if (err != OK) {
 		free(cwd);
-
 		if (err == ERR_HELP) { // Returned by --help and --version, so success.
 			return 0;
 		}
-		return 255;
+		return (int)err;
```

Comment:
This is up for debate. The previous `255` was pretty arbitrary, and inconsistent on Web where we returned the Godot `Error` code.

I chose to unify around returning the `Error` code, as it's marginally useful information. But we could also just return `EXIT_FAILURE` to be consistent with other types of early failures, like when `Main::start()` fails below.

Reply:
Alright, I went ahead and harmonized to `EXIT_FAILURE`.